docs/azmcp-commands.md

```diff
@@ -24,40 +24,69 @@ The following options are available for all commands:
 
 The Azure MCP Server can be started in several different modes depending on how you want to expose the Azure tools:
 
-#### Default Mode
+#### Default Mode (Namespace)
+
+Exposes Azure tools grouped by service namespace. Each Azure service appears as a single namespace-level tool that routes to individual operations internally. This is the default mode to reduce tool count and prevent VS Code from hitting the 128 tool limit.
```

Comment:
nit: consider phrasing this to be more platform-neutral "... and prevent hosts like VS Code, XCode, and JetBrains from hitting max tool limits".